sympy/core/tests/test_expr.py

```diff
@@ -1011,7 +1011,6 @@ def test_extractions():
     assert (Rational(1, 2)*x).extract_multiplicatively(3) == x/6
     assert (sqrt(x)).extract_multiplicatively(x) is None
     assert (sqrt(x)).extract_multiplicatively(1/x) is None
-    assert x.extract_multiplicatively(-x) is None
```

Comment:
i do agree, but i guess as doc in extract_multiplicatively says it returns result if expr can be expressed in terms of c*result and i felt -x*-1=x so it returns -1.